### PR TITLE
Release 20.08.2021

### PR DESCRIPTION
**Pull request by valentinyanakiev**
- Fixed bug where communication sockets were not disposed
- Fixed bug where organisations searched by tagset threw an exception
- Added ordering of the Ecoverses returned by the `ecoverse` query
- Exposed mailslurper through traefik
- Exposed parent Challenge on Opportunity

### CODE REVIEW

**Review by Comoque1 — APPROVED** [review]

File: .build/traefik/http.yml [hunk 1]
```diff
@@ -1,9 +1,14 @@
 http:
   services:
-    ipfs-http-adapter:
```
Comment (techsmyth) [inline .build/traefik/http.yml:3]: Is the ipfs adapter still exposed?

**Review by techsmyth — APPROVED** [review]
Nice set of updates. Only thing that looked odd is the ipfs http adapter?

File: .build/traefik/http.yml (hunk 1 above)
Comment (valentinyanakiev) [reply .build/traefik/http.yml:3]: It is and it must be exposed in order to make IPFS work with Alkemio Server outside of the docker-compose network.

Comment (techsmyth) [reply .build/traefik/http.yml:3]: This PR makes it look like the ipfs adapter is being removed

**Pull request merged**